examples/contract-transfer/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2018"
 [dependencies]
 ink_primitives = { version = "3.0.0-rc3", path = "../../crates/primitives", default-features = false }
 ink_metadata = { version = "3.0.0-rc3", path = "../../crates/metadata", default-features = false, features = ["derive"], optional = true }
-ink_env = { version = "3.0.0-rc3", path = "../../crates/env", default-features = false }
+ink_env = { version = "3.0.0-rc3", path = "../../crates/env", default-features = false, features = [ "ink-debug" ] }
```

Comment:
A better way was to introduce an `ink-debug` feature and forward that to `ink_env`.

Reply:
So the problem with this approach is that `cargo-contract` builds with `--no-default-features`. But there is a ticket to forward an `ink-debug` feature: https://github.com/paritytech/cargo-contract/issues/284.

But for now it means that if somebody builds `contract-transfer` with `cargo-contract` right now we need to explicitly enable the feature for `ink_env` like done in this PR, otherwise the user will not see the debug output.

Reply:
We should raise the priority of the `--debug` flag on `cargo-contract` then.